gslab_scons/builders/build_lyx.py

```diff
@@ -1,4 +1,5 @@
 import os
+import subprocess
 import shutil
 import gslab_scons.misc as misc
```

Comment:
On second thought, I disagree. There are only three Stata only helper functions in misc; this method means we'll be importing four to five helper functions separately and I'm not convinced that will be cleaner than the current implementation.